emcc.py

```diff
@@ -544,6 +536,37 @@ def get_all_js_library_funcs(temp_files):
   return library_fns_list
 
 
+def filter_link_flags(flags, using_lld):
+  def is_supported(f):
+    if using_lld:
+      for flag, takes_arg in UNSUPPORTED_LLD_FLAGS.items():
+        if f.startswith(flag):
+          shared.warning('ignoring unsupported linker flag: `%s`', f)
```

Comment:
I guess this code was just moved around, but maybe this should integrate with the warning manager?

Reply:
I'll do that as part of the outstanding warning manager changes.